feat(android): Add docs for screenshots/vh BeforeCaptureCallback (#7239)
getsentry/sentry-docs: src/platform-includes/enriching-events/attach-screenshots/android.mdx

  <meta-data android:name="io.sentry.attach-screenshot" android:value="true" />
</application>
```

### Customize Screenshot Capturing

<Note>

Requires SDK version `6.24.0` or higher.

</Note>

> On Android capturing screenshots can be an resource-intense operation, thus screenshot capturing is limited to one screenshot every 2 seconds using a debouncing mechanism. This behavior can be overruled by supplying a `BeforeCaptureCallback` for screenshots in the `SentryAndroidOptions`.

Reviewer: ```suggestion
Because capturing screenshots can be a resource-intensive operation on Android, it's limited to one screenshot every 2 seconds using a debouncing mechanism. This behavior can be overruled if you supply a `BeforeCaptureCallback` for screenshots in the `SentryAndroidOptions`.
```

> The `BeforeCaptureCallback` also allows you to customize the behavior based on event data, giving you full control over when a screenshot should be captured and when not.
> 
> For example, screenshot capturing can be limited to only crashed events:

Reviewer: ```suggestion
The `BeforeCaptureCallback` also allows you to customize the behavior based on event data, so you can decide when to capture a screenshot and when not to. For example, you can decide to only capture screenshots of crashed events:
```
